tests/misc/errors/babel-ts/issue-8041.ts

```diff
@@ -1 +0,0 @@
-(a:b)
```

Comment:
babel/parser 7.12 does not throw error. This is probably regression so I reported to babel repo: https://github.com/babel/babel/issues/12180

Reply:
Maybe it's okay to remove this?

```js
**Prettier pr-9408**
[Playground link](https://deploy-preview-9408--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAKAhkgRgShAGhAgAcYBLaAZ2VHQCc6IB3ABXoWpXQBsn0BPaoSx10YANZwYAZWJiyUAObIYdAK5xCcALZY4AE30GAMuiVr0iuADEIdbehjklyEOjUwIBEAAsY27gB1HzJ4SjkwOGkOULIAN1D+VzBKIRAFSjg6GBZRRQdkADMeTMIAK0oADwAhUQkpaXRtOGMFOCKSzRAKyukFRW44AEU1CHgO7lKQOTpMulcsdD1ub2I6BRhAsn0YH2QADgAGQjWITMDRYlc1uDm49sIAR1H4XJJON0oAWig4AwNvHQ4M8yEDcpYCkhipMupltGQVOpYf1BiMxu0oZ1CDAllsdnskAAmbGiMjcfoAYQg2khIFuAFZvGpMgAVJacaFTOIaACSUCMsGkYHWpAAgvzpDB+IMJpkAL5yoA)
```sh
--parser babel
```

**Input:**
```jsx
(a:b)
```

**Output:**
```jsx
(a: b);

```


It didn't crash, original issue https://github.com/prettier/prettier/issues/8041

Reply:
From what I've found out about Flow, that seems to be the correct behavior. `(a:b)` in Flow corresponds to `a as b` in TS. See https://flow.org/en/docs/types/casting/